packages/sync-actions/package.json

```diff
@@ -36,7 +36,7 @@
   },
   "dependencies": {
     "fast-equals": "^2.0.0",
-    "jsondiffpatch": "^0.4.0",
+    "jsondiffpatch": "0.4.0",
```

Comment:
Having the caret in the version is misleading as the semantic versioning does not work for versions in the zero major version.
Even if there's a new zero version (as it is in this case), it won't be used.

The reasoning for that is that zero versions are not considered stable so breaking changes can be expected in minor versions, which is a risk from the consumers point of view.
Actually, it happens with this dependency as version `0.6.0` is a breaking change one.